esp/protocols/http/CMakeLists.txt

```diff
@@ -82,7 +82,8 @@ target_link_libraries(
     jlib
     xmllib
     dalibase
-    thorhelper
+    thorhelper      # this should not be dependent on the engine helper dll
```

Comment:
@afishbeck neither of these dependencies seem particularly correct.  I think the first is for some persistent socket classes, the second for validating a workunit token.  Please check and revisit.

Reply:
Any classes supporting persistent sockets should probably be moved to a more common place. Workunit token validation does check that the workunit is still active.  Will have to look into that.  As you mentioned in the triage meeting, please open a JIRA.

Reply:
HPCC-24920